src/searchForm/overlay/Overlay.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useRef, useState } from 'react'
+import CSSTransition from 'react-transition-group/CSSTransition'
+
+export const enterTransitionDuration = 350
+export const exitTransitionDuration = 500
+export const reducedMotionTransitionDuration = 0
+
+const overlayTimeout = {
+  enter: enterTransitionDuration,
+  exit: exitTransitionDuration,
+}
+
+type OverlayProps = {
+  closeOnBlur: () => void
+  shouldDisplay: boolean
+  children: React.ReactNode
+  className: string
+}
+
+export const Overlay = ({ closeOnBlur, shouldDisplay, children, className }: OverlayProps) => {
+  const overlayRef = useRef<HTMLDivElement>(null)
+  const [reducedMotion, setReducedMotion] = useState(false)
+
+  useEffect(() => {
+    setReducedMotion(window.matchMedia('(prefers-reduced-motion)').matches)
```

Comment:
Maybe have a custom hook for this? I think animations are probably used elsewhere.

Reply:
Done